chapter_optimization/adadelta.ipynb

```diff
@@ -61,6 +61,15 @@
     "%load ../utils/TrainingChapter11.java"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ai.djl.translate.TranslateException;"
```

Comment:
Can you put this with the rest of the imports?

Reply:
This is the only one import in this notebook. We are using separate cell blocks for `%maven`, `%load`, and `import`.